python/cuspatial/cuspatial/core/_column/geocolumn.py

```diff
@@ -179,6 +179,25 @@ def copy(self, deep=True):
         )
         return result
 
+    @property
+    def valid_count(self) -> int:
+        """
+        Returns the number of valid geometries stored in this GeoColumn.
+        Null support is implemented in GeoSeries and this is not expected
+        to ever return less than the length of the GeoColumn. This is
+        provided as a cudf shim layer.
+        """
+        return self._meta.input_types.valid_count
+
+    def has_nulls(self) -> bool:
+        """
+        Returns True if any of the geometries stored in this GeoColumn are
+        null.
+        Null support is implemented in GeoSEries and this is not expected
+        to ever return True, but is provided as a cudf shim layer.
```

Comment:
This seems like the kind of comment that will grow stale. Do we really want to document this expectation?

Also, typo.

```suggestion
        Null support is implemented in GeoSeries and this is not expected
        to ever return True, but is provided as a cudf shim layer.
```